src/StackExchange.Redis/Enums/HashFieldFlags.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace StackExchange.Redis;
+
+/// <summary>
+///  Specifies the options to the HSETF command when to create hash/fields and the return data
+/// </summary>
+[Flags]
```

Comment:
It feels like this is a duplicate re `When`, perhaps coupled with some API shaping - I'd need to think. Either way: we've tried to lean away from abbreviations

Reply:
abbreviations fixed